.github/dependabot.yml

```diff
@@ -4,3 +4,8 @@ updates:
     directory: "/"
     schedule:
       interval: daily
+
+  - package-ecosystem: "github-actions"
```

Comment:
Do the quotes here and on the `internal` below matter, as they are not used in the above configuration?

Reply:
No, copy/paste from the otel-operator, which likely added each one of them at different times :-)